deploy/helm/kuberhealthy/templates/prometheusrule.yml

```
@@ -1,14 +1,21 @@
 {{- if .Values.prometheus.enabled }}
-{{- if .Values.prometheus.enableAlerting }}
+{{- /*
+  Also check for .Values.prometheus.enableAlerting to support backwards compatibility since version 69+
+*/}}
+{{- if or .Values.prometheus.prometheusRule.enabled .Values.prometheus.enableAlerting }}
 ---
 apiVersion: monitoring.coreos.com/v1
 kind: PrometheusRule
 metadata:
   labels:
     prometheus: {{ .Values.prometheus.name }}
+    release: {{ .Values.prometheus.prometheusRule.release }}
     role: alert-rules
+    {{- if .Values.prometheus.prometheusRule.additionalLabels }}
+    {{- toYaml .Values.prometheus.prometheusRule.additionalLabels | nindent 4 }}
+    {{- end }}
   name: {{ template "kuberhealthy.name" . }}
-  namespace: {{ .Release.Namespace }}
+  namespace: {{ .Values.prometheus.prometheusRule.namespace }}
```

```suggestion
  namespace: {{ .Release.Namespace }}
```
i don't think this should be changed?